lms/djangoapps/certificates/api.py

```diff
@@ -141,7 +141,8 @@ def get_certificate_for_user(username, course_key):
         )
     except GeneratedCertificate.DoesNotExist:
         return None
-    return format_certificate_for_user(username, cert)
+
+    return format_certificate_for_user(username, cert) if format_results else cert
```

Comment:
Suggestion: I find this line confusing to read. Maybe something like

if format_results
return format_certificate_for_user(username, cert)
else
return cert

Reply:
Ok! I'll update this.